src/Stratis.Bitcoin.Features.SmartContracts.Tests/Consensus/Rules/AllowedScriptTypeRuleTests.cs

```diff
@@ -81,7 +81,7 @@ public void P2PKHInput_MultiSigOutput_Passes()
 
             Transaction transaction = this.network.CreateTransaction();
             transaction.Inputs.Add(GetP2PKHInput());
-            Script script = PayToMultiSigTemplate.Instance.GenerateScriptPubKey(2, new PubKey[] {new Key().PubKey, new Key().PubKey, new Key().PubKey});
+            Script script = PayToFederationTemplate.Instance.GenerateScriptPubKey(new Stratis.Sidechains.Networks.FederationId(new byte[10]));
```

Comment:
I think we should keep the P2MS test and add one for the new pay-to-fed type

Reply:
Done.